exporter/README.md

```diff
@@ -165,6 +152,18 @@ psql -d template1 -c "CREATE EXTENSION pg_stat_statements;"
 
 ##### Monitoring Setup
 
+| Query File            | Description                                                                                              |
+|-----------------------|----------------------------------------------------------------------------------------------------------|
+| functions_pg10.sql    | Creates ccp_monitoring role with all necessary grants. Creates any extra monitoring functions required.  |
```

Comment:
To be consistent throughout the docs, `ccp_monitoring` should have the backticks around it

Reply:
Fixed